jwala-webapp/src/main/webapp/resources/js/react/group-operations/GroupOperationsDataTable.js

```diff
@@ -843,12 +839,8 @@ var GroupOperationsDataTable = React.createClass({
     jvmConfirmDeleteCallback: function(data) {
         let self = this;
         jvmService.deleteJvm(data.id.id, true).then(function(e){
-            // Remove deleted row
-            $("[id*='jvm-child-table']").find("td:contains('" + data.jvmName + "')").filter(function() {
-                if ($(this).text() === data.jvmName) {
-                    $(this).parent().remove();
-                }
-            });
+            $.alert("JVM " + data.jvmName + " was successfully deleted. Jwala will need to refresh to display the latest data and recompute the states.",
```

Comment:
Again, I wouldn't show the message. Or maybe if you have to reload, can you just refresh the container where the table exists and show the spinning gears?